simulation/setup_dht.cpp

```diff
@@ -182,6 +185,18 @@ struct dht_node final : lt::dht::udp_socket_interface
 		return true;
 	}
 
+	address get_external_address() override
+	{
+		return get_local_address();
```

Comment:
It's just for testing. The local and external addresses only differ if there's NAT and that's not what we're testing here.